checker/match.go

```diff
@@ -193,6 +195,44 @@ func matchHTTPMethods(methods []string, reqMethod string) bool {
 	return false
 }
 
+func matchHTTPPaths(paths []*proto.HTTPMatch_PathMatch, reqPath string) bool {
+	log.WithFields(log.Fields{
+		"paths":   paths,
+		"reqPath": reqPath,
+	}).Debug("Matching HTTP Paths")
+	if len(paths) == 0 {
+		log.Debug("Rule has 0 HTTP Paths, matched.")
+		return true
+	}
+	// Accept only valid paths
+	if !strings.HasPrefix(reqPath, "/") {
+		s := fmt.Sprintf("Invalid HTTP Path \"%s\" from dataplane", reqPath)
```

Comment:
We should be panicking with a type that is unique to this error, so that we will be able to distinguish it when recovering.